pkg/pillar/devicenetwork/dhcpcd.go

```diff
@@ -39,7 +39,7 @@ func UpdateDhcpClient(log *base.LogObject, newConfig, oldConfig types.DevicePort
 		} else {
 			log.Infof("updateDhcpClient: found old %v\n",
 				oldU)
-			if !reflect.DeepEqual(newU.DhcpConfig, oldU.DhcpConfig) {
+			if !reflect.DeepEqual(newU.DhcpConfig, oldU.DhcpConfig) || oldU.HasError() {
```

Comment:
See comment below.

Reply:
When there are port errors in old DPC, we should start/re-start dhcpcd for the new DPC even when there are no differences between old and new DPC. This is for the case when we have transient driver/other issues with ports and/or the ports showing up late. For ex: lte